website/docs/components/text/partials/code/how-to-use.md

```diff
@@ -0,0 +1,136 @@
+The `Text` component is a **code-only** component used to apply specific predefined HDS typographic styles to a block of text or content.
+
+The equivalent in Figma is not a component, but a set of [typographic Figma styles](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=1262-9192).
```

Comment:
This should probably go into an info banner.

Reply:
Done